app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt

```diff
@@ -77,6 +88,10 @@ fun setLongTapTimeout(delay: Int) {
     mDevice.executeShellCommand("settings put secure long_press_timeout $delay")
 }
 
+private fun hideShowJumpBackInCFR(homeActivityTestRule: HomeActivityTestRule) {
+    homeActivityTestRule.activity.settings().shouldShowJumpBackInCFR = false
```

Comment:
nitpick (non-blocking): should we add a comment explaining why this needs to be called in afterActivityLaunched and not before?